src/Osi/OsiSolverInterface.hpp

```diff
@@ -24,6 +25,10 @@
 #include "OsiCollections.hpp"
 #include "OsiSolverParameters.hpp"
 
+#ifdef COINUTILS_HAS_GLPK
+#include "glpk.h"
+#endif
```

Comment:
It shouldn't be necessary to have this in the Osi header file.
In particular, just because the CoinUtils lib was build with Glpk, this doesn't mean that anyone why uses the Osi header will have the Glpk header in its include header path.    
It's not even clear that the Osi build will find the Glpk header. To ensure that, Osi should rather check whether Glpk is available.

Reply:
This is fixed in 1748292.